exercises/palindrome-products/Example.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Palindrome
+{
+    private Palindrome(int value, ISet<Tuple<int, int>> factors)
+    {
+        Value = value;
+        Factors = factors;
+    }
+
+    public int Value { get; }
+
+    public ISet<Tuple<int, int>> Factors { get; }
+
+    public static Palindrome Largest(int minFactor, int maxFactor)
```

Comment:
I noticed that other language tracks allow for `Largest` & `Smallest` to take just a `maxFactor` and default `minFactor` to `1`. Maybe add an overload that does that?